app/models/fabrique_evenement.rb

```diff
@@ -10,7 +10,7 @@ def initialize(parametres)
   def call
     partie = recupere_partie
     evenement = Evenement.new EvenementParams.from(parametres).merge(partie: partie)
-    CreeEvenementAction.new(partie, evenement).call
+    evenement.save
```

Comment:
Ce que je te propose c'est de supprimer le concern et l'ancienne classe `CreeEvenementAction`

et donner la logique métier de :
1. Créer un évènement (en base de donnée)
2. Programmer un job pour persister les métriques si c'est le dernier évèneement  

```suggestion
    evenement.save
    PersisteMetriquesPartieJob.perform_later(partie) if evenement.nom == FIN_SITUATION
```

On évite ainsi d'utiliser les callbacks de Rails permettant de garder le model Evenement plus unitaire